project/DataModule.py

```diff
@@ -125,9 +111,8 @@ def get_labels(self, data) -> list:
             if sample["data_label"] == "label"
         ]
 
-    def prepare_data(self, *args, **kwargs):
+    def setup(self, *args, **kwargs):
```

Comment:
I would like to have an in-depth discussion around setup() versus prepare_data(), potentially at the next TT.

Reply:
Keep as is for now, but explore in the future.